static/js/ducks/sources.js

```diff
@@ -58,6 +67,25 @@ export function fetchFavoriteSources(filterParams = {}) {
   return API.GET("/api/sources", FETCH_FAVORITE_SOURCES, filterParams);
 }
 
+export function fetchGcnEventSources(dateobs = null, filterParams = {}) {
+  addFilterParamDefaults(filterParams);
+  filterParams.localizationDateobs = dateobs;
+  filterParams.localizationCumprob = 0.95;
+
+  if (dateobs) {
+    filterParams.startDate = dayjs(dateobs).format("YYYY-MM-DD HH:mm:ss");
+    filterParams.endDate = dayjs(dateobs)
+      .add(7, "day")
+      .format("YYYY-MM-DD HH:mm:ss");
+  }
+
+  filterParams.startDate = null;
```

Comment:
Isn't this just wiping out the values set above?

Reply:
Commenting this (Debugging line) out!